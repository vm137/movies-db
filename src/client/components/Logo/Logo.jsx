import React from 'react';
import './Logo.scss';

const Logo = () => (
  <p className="logo">netflixroulette</p>
);

export default Logo;
