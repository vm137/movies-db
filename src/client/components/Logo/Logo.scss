.logo, .logo-bottom {
    color: #ff594b;
    font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
    font-weight: bold;
    font-size: 17px;
    text-shadow: 1px 1px 2px rgb(69, 69, 69);
}

.logo {
    top: 16px;
    left: 5%;
    position: absolute;
}

.logo-bottom {
    top: 18px;
    left: 5%;
    position: absolute;
}
