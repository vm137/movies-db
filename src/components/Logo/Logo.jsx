import React from 'react';

const Logo = () => (
  <p className="logo">netflixroulette</p>
);

export default Logo;
