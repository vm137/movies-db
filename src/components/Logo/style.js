import injectSheet from 'react-jss';

const shared = {
  color: '#ff594b',
  fontFamily: 'Arial, "Helvetica Neue", Helvetica, sans-serif',
  fontWeight: 'bold',
  fontSize: '17px',
  textShadow: '1px 1px 2px rgb(69, 69, 69)',
};

const styles = {
  logo: {
    ...shared,
    top: '16px',
    left: '5%',
    position: 'absolute',
  },
  logoBottom: {
    top: '18px',
    left: '5%',
    position: 'absolute',
  },
};

export default injectSheet(styles);
