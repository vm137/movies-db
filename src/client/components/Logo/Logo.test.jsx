import { render } from 'enzyme';
import React from 'react';

import Logo from './Logo';

describe('Logo', () => {
  it('renders matching snapshot', () => {
    const wrapper = render(<Logo />);

    expect(wrapper).toMatchSnapshot();
  });
});
