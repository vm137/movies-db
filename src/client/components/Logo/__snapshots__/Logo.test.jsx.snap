// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders matching snapshot 1`] = `ShallowWrapper {}`;
